.editorconfig

```diff
@@ -0,0 +1,24 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+# WordPress Coding Standards
+# https://make.wordpress.org/core/handbook/coding-standards/
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+tab_width = 4
+indent_style = tab
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.txt]
+trim_trailing_whitespace = false
+
+[*.{md,json,yml}]
+trim_trailing_whitespace = false
+indent_style = space
+indent_size = 2
```

Comment:
This is incorrect - per WPCS, JSON should be intended by a 4-space tab.

See https://core.trac.wordpress.org/browser/trunk/.editorconfig or https://github.com/Parsely/wp-parsely/blob/develop/.editorconfig for the best examples (they also avoid re-stating the defaults for `[*]` as well).